extension/src/shared/view/selection/OpenDataLinkSection.tsx

```diff
@@ -0,0 +1,42 @@
+import { Divider, Button, styled } from "@mui/material";
+import { type FC } from "react";
+
+import { InspectorItem } from "../../../prototypes/ui-components";
+import { DatasetFragmentFragment } from "../../../shared/graphql/types/catalog";
+
+export interface DatasetProps {
+  dataset?: DatasetFragmentFragment;
+}
+
+export const OpenDataLink: FC<DatasetProps> = ({ dataset }) => {
+  const openDataUrl = dataset?.openDataUrl;
+  const hasDataset = !!openDataUrl;
+
+  return (
+    <>
+      {hasDataset && (
+        <>
+          <Divider />
+          <InspectorItem>
+            <CustomButton
+              size="small"
+              variant="outlined"
+              fullWidth
+              onClick={() => {
+                if (openDataUrl) {
+                  window.open(openDataUrl, "_blank");
+                }
+              }}>
+              オープンデータを入手
+            </CustomButton>
```

Comment:
```suggestion
            <CustomButton
              size="small"
              variant="outlined"
              fullWidth
              href={openDataUrl}
              target="_blank"
              >
              オープンデータを入手
            </CustomButton>
```

Reply:
when i put target attribute on customButton, it has typeError ts(2322)
i think that creating custom Buttons using styled requires a different approach, as it does not directly support href and target attributes

for example,
custom a tag or 
```
<Button
size="small"
variant="outlined"
fullWidth
component="a"
href={openDataUrl}
target="_blank"
sx={{
     borderRadius: "2px",
     borderColor: "#D9D9D9",
}}>
     オープンデータを入手
</Button>
```